packages/electron/README.md

```diff
@@ -0,0 +1,504 @@
+
+```ts
+const client = BacktraceClient.initialize({
+    // ignoring all but database config for simplicity
+    database: {
+        enable: true,
+        path: '/path/to/the/database/directory',
+        captureNativeCrashes: true,
+    },
+});
+
+// manually send and keep the data on connection issue
+client.database.send();
+// manually send and remove all data no matter if received success or not.
+client.database.flush();
+```
+
+#### Database Configuration
+
+| Option Name               | Type    | Description                                                                                                                                                                  | Default | Required?                |
+| ------------------------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- | ------------------------ |
+| `enabled`                 | Boolean | Enable/disable offline database support.                                                                                                                                     | false   | <ul><li>- [x] </li></ul> |
+| `path`                    | String  | Local storage path for crash data.                                                                                                                                           | -       | <ul><li>- [x] </li></ul> |
+| `createDatabaseDirectory` | Boolean | Allow the SDK to create the offline database directory.                                                                                                                      | true    |
+| `autoSend`                | Boolean | Sends reports to the server based on the retry settings. If the value is set to 'false', you can use the Flush or Send methods as an alternative.                            | true    |
+| `maximumNumberOfRecords`  | Number  | The maximum number of reports stored in the offline database. When the limit is reached, the oldest reports are removed. If the value is equal to '0', then no limit is set. | 8       |
+| `retryInterval`           | Number  | The amount of time (in ms) to wait between retries if the database is unable to send a report.                                                                               | 60 000  |
+| `maximumRetries`          | Number  | The maximum number of retries to attempt if the database is unable to send a report.                                                                                         | 3       |
+| `captureNativeCrashes`    | Boolean | Capture and symbolicate stack traces for native crashes if the runtime supports this. A crash report is generated, stored locally, and uploaded upon next start.             | false   |
+
+#### Native crash support
+
+The Backtrace Electron SDK can capture native crashes generated by a Electron application such as Assert/OOM crashes. In
```

Comment:
We do collect all native crashes. Not only OOMs/Asserts. TO be honest, if we don't have enough luck, we won't collect ooms, or they will be visible as a regular crash (failed to allocate memory or something similar)

Reply:
>such as

This does not limit the crashes to these options only.

Reply:
can we remove such as? and just focus on we're capturing crashes?

Reply:
Done